test.sh

```diff
@@ -99,7 +99,7 @@ function tear_down() {
 
 function reset_containers() {
     docker_compose stop -t 0 &>> ${LOGFILE};
-    docker_compose rm -vf &>> ${LOGFILE};
+    docker_compose rm -f &>> ${LOGFILE};
```

Comment:
If we don't rely on volumes for the caching, this change isn't needed for the stated purpose of this pull request. (Maybe it's a good idea anyway, though.)